x/bank/v2/keeper/keeper.go

```diff
@@ -252,3 +258,18 @@ func newBalancesIndexes(sb *collections.SchemaBuilder) BalancesIndexes {
 type BalancesIndexes struct {
 	Denom *indexes.ReversePair[[]byte, string, math.Int]
 }
+
+// AppendSendRestriction adds the provided SendRestrictionFn to run after previously provided restrictions.
+func (k Keeper) AppendSendRestriction(restriction types.SendRestrictionFn) {
```

Comment:
nit: can be move this in another file, and call the API Global?

Reply:
That's fine, I meant renaming to `AppendGlobalSendRestriction` and maybe put all sendrestrictions logic ousitde of keeper.go but still in the keeper